### DISCUSSION TITLE
Issue running on Debian 12 bookworm.

### DISCUSSION THREAD

**Discussion by malnajdi** [typegoose/mongodb-memory-server#757] [Q&A]
Hey team, i am not sure if there is a binary that is supported for debian 12 bookworm or not. If there is any, kindly help me find it.

**Comment by hasezoey** [comment]
there is no binary for debian12 yet, do debian11 binaries work?

**Reply by malnajdi** [reply]
nope.

**Reply by hasezoey** [reply]
> nope.

please post the error you are having, any missing libraries?

**Reply by malnajdi** [reply]
dpkg: dependency problems prevent configuration of mongodb-org-server:
 mongodb-org-server depends on libssl1.1 (>= 1.1.1); however:
  Package libssl1.1 is not installed.


I could build it from scratch but, i am using docker and running my tests inside it now.

**Reply by hasezoey** [reply]
> dpkg: dependency problems prevent configuration of mongodb-org-server:

i have no clue what this has to do with mongodb-memory-server, that looks like a package directly provided by mongodb or some kind of ppa

but assuming the binaries downloaded with MMS for debian 11 on 12 will result in the same error, could you try using ubuntu binaries? (`MONGOMS_ARCHIVE_NAME=mongodb-linux-x86_64-ubuntu2004-6.0.4.tgz` or `MONGOMS_ARCHIVE_NAME=mongodb-linux-x86_64-ubuntu2204-6.0.4.tgz`)

**Reply by malnajdi** [reply]
```
 Starting the MongoMemoryReplSet Instance failed, enable debug log for more information. Error:
StdoutInstanceError: Instance failed to start because a library is missing or cannot be opened: "libcrypto.so.1.1"
```
